tasks_downstream.py

```diff
@@ -24,7 +24,21 @@
 
 PROJECT_ROOT = Path(__file__).parent.absolute()
 SRC_PATH = PROJECT_ROOT / "odoo" / "custom" / "src"
-UID_ENV = {"GID": str(os.getgid()), "UID": str(os.getuid()), "UMASK": "27"}
+UID_ENV = {
+    "GID": os.environ.get("DOODBA_GID", str(os.getgid())),
```

Comment:
BTW, why did you add this option? It seems ok, but I don't see you using it anywhere neither in the code nor in the documentation.

Reply:
As it was before, even if you had exported `$GID` somewhere, it wouldn't work because it would be always overriden by `os.getgid()`. Same for UID.

So I realized that it seems more safe to prefix all the env vars with `DOODBA_`, because GID, UID and UMASK are so common names that I fear they could collide somewhere else.

In docker, container UID/GID are the same, but in podman they are mapped using subuid and subgid mechanism and can easily lead to permission problems.